docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -161,6 +158,41 @@ new MVTLayer({
 })
 ```
 
+* `getVisibleTiles` (Function)
```

Comment:
We need to explain better this method, we need to explain what the tile object has inside. Something like [this](https://deck.gl/docs/api-reference/geo-layers/tile-layer#gettiledata)